templates/android/library/src/main/java/io/appwrite/Permissions.kt.twig

```diff
@@ -0,0 +1,21 @@
+package {{ sdk.namespace | caseDot }}
+
+class Permissions {
+    companion object {
+        fun read(role: String): String {
```

Comment:
```suggestion
        fun read(role: Role): String {
```
Its better to keep `role` of type Role to ensure stronger typing and better ID completion...